cmd/manager/main.go

```diff
@@ -126,13 +128,20 @@ func main() {
 	// work
 	stopCh := signals.SetupSignalHandler()
 
+	accountWatcherClient, err := client.New(cfg, client.Options{})
```

Comment:
What are the implications of having two clients?

Reply:
@fahlmant since we are using this only to list secrets once every 15 minutes I don't expect their to be a big issue.